docs/native/src/overview.rst

```diff
@@ -1,5 +1,76 @@
+
+..  note::
+    * MSA Free: the user has a Microsoft account, but no O365 subscription
+    * MSA Subscription: the user has a Microsoft account and an individual O365 subscription
+    * Org subscription: the user belongs to an organization which has an O365 subscription
+    * Premium editing: refers to certain Office for iOS edit functionality which requires an O365 subscription to use
+
+================= ===============  ===============   ===============
+ACCOUNT TYPE      Consumer         Enterprise        EDU
+================= ===============  ===============   ===============
+Not signed in     View only	       View only	       Free edit
+MSA Free          Free Edit	       View only	       Free edit
+MSA Subscription  Premium editing  Premium editing	 Premium editing
+Org Subscription  Premium editing  Premium editing	 Premium editing
+================= ===============  ===============   =============== 
+
+
+Integration process
+-------------------
+
+You can use the Web Application Open Platform Interface (WOPI) protocol to integrate Office for iOS with your application. The WOPI protocol enables Office for iOS to access and change files that are stored in your service.
+
+To integrate your application with Office for iOS, you need to do the following:
+ #. Become a member of the Office 365 - Cloud Storage Partner Program. Currently integration with Office for iOS using WOPI is only available to cloud storage partners. You can learn more about the program, as well as how to apply, at `Office 365 Cloud Storage Partner Program. <http://dev.office.com/programs/officecloudstorage>`_
+ #. Provide the required on-boarding information as described in the section titled :ref:`Onboarding information <onboarding>`.
+ #. Obtain your ProviderId and app store URLs from Microsoft. The app store URLs are the URLs you should use to launch Office on a platform's app store. 
+ #. Add your domain to the WOPI domain :ref:`allow list <allow list>`. This is required even if you do not integrate with Office Online so you can verify your WOPI integration with the :ref:`validator app <Validator app>`.
+ #. Implement the WOPI protocol - a set of REST endpoints that expose information about the documents that you want to view or edit in Office for iOS. The set of WOPI operations that must be supported is described in the section titled :ref:`WOPI implementation requirements for Office for iOS integration <requirements>`.
+ #. Implement required changes to your app, including all applicable promotion requirements, as well as :ref:`opening files from your app into Office <OpenFilesiOSOffice>`. Contact your Office for iOS integration contacts to receive directions to run Office for iOS in test mode, to enable testing. 
+ #. Run the :ref:`validator app <Validator app>` and fix any issues until the validator reports a 100% pass rate. 
```

Comment:
I think this is probably not the right place for this level of detail; I'd move the details off into a separate section and link to it, especially the testing guidelines and the shipping process.